src/args/lint-with-ajv/ajv-cli/package.json

```diff
@@ -1,5 +1,6 @@
 {
   "dependencies": {
-    "ajv-cli": "*"
+    "ajv-cli": "*",
+    "fast-json-patch": "3.1.1"
```

Comment:
@dacevedo12 Is this what do we do when we want to update a vulnerable sub-dependency?

Reply:
as can be seen in the package lock diff line 293, this didn't have the intended effect.

it still installs fast-json-patch 2.2.1

you can use overrides to force npm to install the version you desire

Reply:
@rohaquinlop

Reply:
more info here https://docs.npmjs.com/cli/v10/configuring-npm/package-json#overrides

Reply:
Thanks!!!